exonum-java-binding/CHANGELOG.md

```diff
@@ -168,7 +170,8 @@ Parent module and BOM module were released as they are required dependencies to
 
 The first release of Exonum Java Binding.
 
-[Unreleased]: https://github.com/exonum/exonum-java-binding/compare/v0.4...HEAD
+[Unreleased]: https://github.com/exonum/exonum-java-binding/compare/v0.5...HEAD
+[0.4]: https://github.com/exonum/exonum-java-binding/compare/v0.4...v0.5
```

Comment:
We plan to use `ejb/v0.5.0` tag to distinguish ejb releases from jlc ones, don't we?